### PR TITLE
Explain timer pin map command structure

### PR DESCRIPTION
**Pull request by howels**
I have trouble understanding the new TIMER_PIN_MAP commands and perhaps others do as well, so I have added some explanation. This is written by someone with a loose understanding but I hope it adds more information for others.

### CODE REVIEW

**Comment by haslinghuis** [comment]
@howels please also fix the typo in the link name (crearing -> creating)

File: docs/manufacturer/crearing-configuration.mdx [hunk 1]
```diff
@@ -115,6 +115,15 @@ To enable the `timer map` command you need to add the define `USE_TIMER_MAP_PRIN
 
     make TARGET EXTRA_FLAGS="-DUSE_TIMER_MAP_PRINT"
 
+The `TIMER_PIN_MAP` config statement is broken down as follows:
+
+`TIMER_PIN_MAP(WW, PXX, YY, ZZ)`
```
Comment (haslinghuis) [inline docs/manufacturer/crearing-configuration.mdx:120]:
```suggestion
`TIMER_PIN_MAP(WW, XX, YY, ZZ)`
```

File: docs/manufacturer/crearing-configuration.mdx [hunk 2]
```diff
@@ -115,6 +115,15 @@ To enable the `timer map` command you need to add the define `USE_TIMER_MAP_PRIN
...
+
+* WW - zero-indexed counter, increment by 1 for each line in the timer map
+* XX - pin number to set the timer on
```
Comment (haslinghuis) [inline docs/manufacturer/crearing-configuration.mdx:123]:
```suggestion
* XX - one indexed occurrence of pin number to set the timer on based on AT32 or STM32 timer definitions
```

File: docs/manufacturer/crearing-configuration.mdx [hunk 3]
```diff
@@ -115,6 +115,15 @@ To enable the `timer map` command you need to add the define `USE_TIMER_MAP_PRIN
...
+* YY - selected instance of the DMA configuration where multiple options are available see your MCU's target timer code for the list of options. For example F772 should reference [this timer file](https://github.com/betaflight/betaflight/blob/master/src/main/drivers/stm32/timer_stm32f7xx.c).
```
Comment (haslinghuis) [inline docs/manufacturer/crearing-configuration.mdx:124]:
```suggestion
* YY - selected instance of the timer configuration where multiple options are available. See your MCU's target timer code for the list of options. For example F772 should reference [this timer file](https://github.com/betaflight/betaflight/blob/master/src/main/drivers/stm32/timer_stm32f7xx.c).
```

Comment (howels) [reply docs/manufacturer/crearing-configuration.mdx]: Renamed file and included changes

**Review by haslinghuis — APPROVED** [review]

**Review by SupaflyFPV — APPROVED** [review]

**Pull request merged**